lib/graphql/generators/templates/graphqls_controller.erb

```diff
@@ -0,0 +1,30 @@
+class GraphqlsController < ApplicationController
+  def create
+    variables = ensure_hash(params[:variables])
+    query = params[:query]
+    context = {
+      # Query context goes here, for example:
+      # current_user: current_user,
+    }
+    result = <%= schema_name %>.execute(query, variables: variables, context: context)
+    render json: result
+  end
+
+  private
+
+  # Handle form data, JSON body, or a blank value
+  def ensure_hash(ambiguous_param)
+    case ambiguous_param
+    when String
+      if ambiguous_param.present?
+        JSON.parse(ambiguous_param)
```

Comment:
I guess we could assert that the parsed value is a Hash. 

I just realized that this won't work on Rails 5 since ActionController::Parameters won't match `when Hash` 😖 

I'll take a look at whether I can return the parameters as-is or whether they need to be `to_unsafe_hash`ed.